include/miopen/miopen.h

```diff
@@ -337,8 +337,24 @@ typedef enum
     miopenBFloat16 = 5, /*!< 16-bit binary floating point (8-bit exponent, 7-bit fraction)
                            (Partially supported) */
     miopenDouble = 6,   /*!< 64-bit floating point (Partially supported) */
+    miopenHalfx4 = 7,   /*!<Pack of 4 16-bit floating points in NCHW_VECT_C format*/
+    miopenHalfx8 = 8,   /*!<Pack of 8 16-bit floating points in NCHW_VECT_C format*/
 } miopenDataType_t;
 
+/*! @ingroup tensor
+ * @enum miopenTensorLayout_t
+ * MIOpen tensor layouts. NCHW, NHWC, CHWN and NCHW_VECT_C with certain vector length(vect_c=4,
+ * vect_c=8) are supported in MIOpen.
+ */
+typedef enum
+{
+    miopenTensorNCHW        = 0, /*!< NCHW memory layout (Fully supported) */
+    miopenTensorNHWC        = 1, /*!< NHWC memory layout (Fully supported) */
+    miopenTensorCHWN        = 2, /*!< CHWN memory layout (Not supported) */
+    miopenTensorNCHW_VECT_C = 3, /*!< NCHW_VECT_C memory layout (Partially supported) */
+    miopenTensorCHWN_VECT_C = 4, /*!< CHWC_VECT_C memory layout (Partially supported) */
```

Comment:
The vectorized dimension (C in this case) must be multiple of the pack size (of the packed datatype, i.e. 4 or 8). Please add this restriction to the documentation.

Reply:
Understand.